DP02_09_Custom_Coadds/DP02_09b_Custom_Coadd_Sources.ipynb

```diff
@@ -345,371 +336,329 @@
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "06a83779-a435-41f9-ad18-57c542ca6abb",
+   "id": "16f3600d-17c0-4790-bc87-56490109a7e7",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "schema = afwTable.SourceTable.makeMinimalSchema()\n",
-    "print(schema)"
+    "## 2.1. Set up an output collection for the source detection results"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "100c3ed7-6665-41d7-ba70-07a1939db890",
+   "id": "71fa51a3-7883-4baa-84df-6742b84a9e68",
    "metadata": {},
    "source": [
-    "Create a container, `algMetadata`, which will be used to record metadata about algorithm execution."
+    "First, define an output collection name into which our custom coadd source detection results will be persisted:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "a5cbd008-202d-4c21-a582-3a95a90afe09",
+   "id": "714ae833-e6dc-4362-a140-ae2d20b5ef1f",
    "metadata": {},
    "outputs": [],
    "source": [
-    "algMetadata = dafBase.PropertyList()\n",
-    "print('algMetadata: ')\n",
-    "algMetadata"
+    "my_outputCollection_identifier = 'custom_coadd_window1_test1_nbdet'\n",
+    "my_outputCollection = 'u/' + my_username + '/'+ my_outputCollection_identifier\n",
+    "print(my_outputCollection)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "ea28b4f6-7ccb-4ae4-8f76-4334ccb06c89",
+   "id": "535d5c75-ccb9-4ce4-8986-31dd79a98f99",
    "metadata": {},
    "source": [
-    "## 2.2. Configure tasks\n",
-    "\n",
-    "Configure the task to characterize the image. \n",
-    "Use just one iteration to determine the point spread function (PSF)."
+    "As in the DP0.2 custom coaddition tutorial, the next step is to define a \"simple butler\" object that will be needed for running the SimplePipelineExecutor:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "4c69c185-cea1-4c38-a1ae-41b42719cd7f",
+   "id": "03e56251-fc55-4214-a40d-46d70d0c1f05",
    "metadata": {},
    "outputs": [],
    "source": [
-    "config = CharacterizeImageTask.ConfigClass()\n",
-    "config.psfIterations = 1\n",
-    "charImageTask = CharacterizeImageTask(config=config)"
+    "simpleButler = SimplePipelineExecutor.prep_butler(butler_repo, \n",
+    "                                                  inputs=[inputCollection], \n",
+    "                                                  output=my_outputCollection)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "b6236c37-1cba-4bcf-8a08-7ff6caeb4e59",
+   "id": "0e256101-cfeb-4691-8ce8-ae676c83219d",
    "metadata": {},
    "source": [
-    "Configure the task to detect sources.\n",
-    "Set the threshold that defines what \"detected\" means to be 10 times above the standard deviation (i.e., a signal-to-noise ratio of 10)."
+    "Now check the collection chain associated with your source detection output collection:"
```

Comment:
Re. "Now check the collection chain..." we should specify check for what. I think the user should check that the new output collection appears first? Probably worth adding that a timestamp gets added, as they will see it.

Reply:
Good catch. This material was in notebook 9a but I didn't previously transfer it into the revised version of notebook 9b. I've now added these two items to 9b in commit [f264ae4](https://github.com/rubin-dp0/tutorial-notebooks/pull/161/commits/f264ae4430bde54200bf32463c2fa06479c2537a), which mostly just transfers the relevant cell of notebook 9a into the new version of notebook 9b.